### ISSUE TITLE
What is the necessary configuration to deploy the various operators?

### ISSUE THREAD

**Issue by rydevops** [dapr-sandbox/dapr-kubernetes-operator#18]
After installing this within CodeReady Containers, I am not seeing any of the operators available. If I create a deployment with the dapr annotations, I also don't see the sidecar being injected. The only thing I seem to be able to do is created instances of the CRDs. Are there any configurations for this operator?

**Comment by lburgazzoli** [comment]
@rydevops yes, at this stage, you first have to create a DaprControlPlane resource which would trigger the deployment of all the Dapr components. 

I'm working to some docs so that should be clear in a future release.

**Comment by lburgazzoli** [comment]
@rydevops the README has been update, let me know if anything is still missing.

**Comment by lburgazzoli** [comment]
closed as completed by https://github.com/dapr-sandbox/dapr-kubernetes-operator/pull/23 and https://github.com/dapr-sandbox/dapr-kubernetes-operator/pull/39
